### ISSUE TITLE
VisibleOnce Visibility Variant does not work with Nuxt3 SSR / Nuxt Link

### ISSUE THREAD

**Issue by CptJJ** [vueuse/motion#134]
When navigating via a nuxt link component div will not appear the second time around. A reload fixes the issue can all styled divs then appear correctly.

**Comment by foxilated** [comment]
I came here to submit this exact issue, I've been having much trouble with it.

**Comment by cosmincartas** [comment]
I'm having the same issue but not only on `visibleOnce` variant, on `visible` as well. As a temporary fix I'm using the `useMotion` composable which seems to do the job.
```typescript
const motionVariants: MotionVariants = {
    initial: {
        y: 100,
        opacity: 0
    },
    visibleOnce: {
        y: 0,
        opacity: 1
    }
};
const animatedSectionRef = ref<HTMLElement | null>();
useMotion(animatedSectionRef, motionVariants);
```

**Comment by barbecue** [comment]
+

**Comment by fabienbergerat** [comment]
I'm having the same issue but unfortunately using the composable doesn't seem to fix it.

**Comment by BobbieGoede** [comment]
This issue is likely fixed by #171 which is now included in the latest release v2.1.0, but without a reproduction I'm unable to confirm this, can someone check if this issue has been resolved after installing the latest version?
